### PR TITLE
Opened K5 public map API so it's actually publicly accessible

### PR DESCRIPTION
**Pull request by aoby**

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
lgtm

File: dashboard/app/controllers/api/v1/pd/workshops_controller.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def index
   end
 
   def k5_public_map_index
-    @workshops = @workshops.where(
+    @workshops = Pd::Workshop.where(
```
Comment (ashercodeorg) [inline dashboard/app/controllers/api/v1/pd/workshops_controller.rb:14]: Given that we are opening this up, does it make sense to whitelist the fields being returned to the map?

Comment (aoby) [inline dashboard/app/controllers/api/v1/pd/workshops_controller.rb:14]: It was meant to be public all along, but yeah that's a fair point. I'll prepare another PR with a limited serializer.

**Comment by tanyaparker** [comment]
lgtm

File: dashboard/app/controllers/api/v1/pd/workshops_controller.rb (hunk 1 above)
Comment (aoby) [inline dashboard/app/controllers/api/v1/pd/workshops_controller.rb:14]: Oh, never mind. I already did that :)

This [serializes](https://github.com/code-dot-org/code-dot-org/blob/bd8f28007a2ae38647fcbc9d2a13d2d84d9ed956/dashboard/app/controllers/api/v1/pd/workshops_controller.rb#L19-L19) with the [WorkshopK5MapSerializer](https://github.com/code-dot-org/code-dot-org/blob/7d452d6bb463a0c6368e7384d3da27166bc98361/dashboard/app/serializers/api/v1/pd/workshop_k5_map_serializer.rb) that only displays `:id, :location_name, :location_address, :sessions`

**Pull request merged**